syft/frameworks/torch/hook/hook.py

```diff
@@ -412,15 +412,15 @@ def _hook_torch_module(self):
                 if "native_" in func or f"native_{func}" in dir(torch_module):
                     continue
 
-                self._perform_function_overloading(torch_module, func)
+                self._perform_function_overloading(module_name, torch_module, func)
 
     @classmethod
-    def _get_hooked_func(cls, attr):
+    def _get_hooked_func(cls, module_name, attr):
```

Comment:
nit, can we change this `public_module_name` or something? `module_name` is ambiguous in the context of the problem we were facing with tf